### PR TITLE
CI: .md files changes should not trigger tests

### PR DESCRIPTION
**Pull request by apostasie**

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -6,6 +6,9 @@ on:
       - main
       - 'release/**'
   pull_request:
+    paths-ignore:
+      - '**/*.md'
```
Comment (sondavidb) [inline .github/workflows/test.yml:10]: I think you can just do '**.md' to account for all markdown files in the project

Comment (apostasie) [reply .github/workflows/test.yml]: Always confused about globs - where they work which way.
Pushing now.
Thanks!

**Comment by apostasie** [comment]
@sondavidb fixed

**Review by AkihiroSuda — APPROVED** [review]
Thanks

**Pull request merged**